docs/source/api_reference/transforms.rst

```diff
@@ -135,14 +135,16 @@ Scaling transforms:
    MinMaxScalerTransform
    MaxAbsScalerTransform
 
-Functional transforms:
+Functional transforms and their utils:
 
 .. autosummary::
    :toctree: api/
    :template: class.rst
 
    LambdaTransform
    AddConstTransform
+   BinaryOperationTransform
+   BinaryOperator
```

Comment:
This doesn't show because it can't be importer from `etna.transforms`. It will be shown if you use `math.BinaryOperator`.